src/pages/Faq/Faq.js

```diff
@@ -1,7 +1,180 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import variables from "styles/variables";
+import ballon from "asset/images/talk-ballon.png";
+import FaqModal from "./FaqModal";
 
 const Faq = () => {
-  return <div>여기는 자주하는 질문</div>;
+  const [faqData, setFaqData] = useState([{}]);
+  const [isModal, setIsModal] = useState(false);
+
+  useEffect(() => {
+    fetch("data/faq.json")
+      .then((res) => res.json())
+      .then((data) => setFaqData(data));
+  }, []);
+
+  const handleModal = () => {
+    setIsModal(!isModal);
```

Comment:
```suggestion
    setIsModal(prev => !prev);
```
boolean 값을 업데이트 해줄 때는 함수형 업데이트로 구현하는 것이 더 안전한 방법입니다.